scripts/create_test_data.py

```diff
@@ -194,6 +200,47 @@ def create_test_data(db: orm.Session) -> FidesUser:
                     phone_number="+1 234 567 8910",
                 ),
             )
+            for action_type in [
+                AuditLogAction.approved,
+                AuditLogAction.denied,
+                AuditLogAction.finished,
```

Comment:
i know this is dummy data, but maybe switching the order of approved and denied might be useful, so in a demo it doesn't look like it was denied and then it finished